docs/dev/install.rst

```diff
@@ -86,15 +86,28 @@ Installation
 
    - ``SITE_URL`` should be set to the URL you will use to connect to your
      local development site. Some people prefer to use
-     ``http://127.0.0.1:8000`` instead of ``localhost``.
+     ``http://127.0.0.1:8000`` instead of ``localhost``. However, should you
+     decide to change the ``SITE_URL``, you also need to
+     [request](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Firefox_Accounts/Introduction)
```

Comment:
I think we should mention that our demo/intro site (http://localhost:8000/intro) will require change of base url if someone will decide to change SITE_URL.